plugins/com.google.cloud.tools.eclipse.appengine.validation.test/src/com/google/cloud/tools/eclipse/appengine/validation/AppEngineWebProblemsTest.java

```diff
@@ -25,48 +25,48 @@
 import org.junit.Assert;
 import org.junit.Test;
 
-public class AppEngineWebBlacklistTest {
+public class AppEngineWebProblemsTest {
   
   @Test
   public void testBlacklisted() {
-    Assert.assertTrue(AppEngineWebBlacklist.contains("application"));
+    Assert.assertTrue(AppEngineWebProblems.contains("application"));
   }
  
   @Test
   public void testContains() {
-    assertFalse(AppEngineWebBlacklist.contains("foo"));
+    assertFalse(AppEngineWebProblems.contains("foo"));
   }
   
   @Test
   public void testContains_nullArg() {
-    assertFalse(AppEngineWebBlacklist.contains(null));
+    assertFalse(AppEngineWebProblems.contains(null));
   }
   
   @Test(expected = NullPointerException.class)
   public void testGetBlacklistElementMessage_nullArg() {
-    AppEngineWebBlacklist.getBlacklistElementMessage(null);
+    AppEngineWebProblems.getDeprecatedElementMessage(null);
   }
   
   @Test(expected = IllegalArgumentException.class)
   public void testGetBlacklistElementMessage_elementNotInBlacklist() {
```

Comment:
The method name contains `Blacklist`. Actually, there are many other such methods.